src/index.html

```diff
@@ -3,12 +3,59 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>flexbox</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="home__link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link  is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops & computers"
+               class="nav__link"
+               data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a href="#laptops & computers"
               class="nav__link"
               data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Tag’s closing bracket should be on the same level as opening one.
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/d6497214-8ad0-4bec-b44c-3953ed113320)